frontend/models/product-list/index.ts

```diff
@@ -61,12 +61,11 @@ export async function findProductList(
    ]);
 
    const productList = result.productLists?.data?.[0]?.attributes;
-
    if (productList == null && deviceWiki == null) {
       return null;
    }
 
-   const deviceTitle = productList?.deviceTitle ?? filterDeviceTitle;
+   const deviceTitle = deviceWiki?.deviceTitle ?? productList?.deviceTitle;
```

Comment:
```suggestion
   const deviceTitle = deviceWiki?.deviceTitle ?? productList?.deviceTitle ?? null;
```

Reply:
This is cleaner so I'm not complaining but I think the previous version still worked